src/scss/index.scss

```diff
@@ -283,35 +288,26 @@ table.field-list {
     }
   }
 
+  // For page headers with toctrees under them
+  li.page-header {
```

Comment:
This just simplifies the CSS a little bit - we were using a bunch of level-specific selectors for our rules, but I think we can get the same effect with the generic rules here

Reply:
Do you remember where/when this "page-header" class is used? I don't directly find it in the output HTML somewhere

Reply:
@choldgraf I don't see this "page-header" class use anywhere, so for now going to remove it again. If it actually had a purpose, please explain and I will add it back :)

Reply:
go for it - I have no idea what I was thinking here because this PR is from 7 months ago :-)